### PR TITLE
Migrate project to Flask

### PR DESCRIPTION
**Pull request by jpmartinspt**
Migrate from django to flask.

# QA
- Set a `.env.local` as described in the README
- `./run`
- Make sure everything still works

Fixes https://github.com/canonical-web-and-design/base-squad/issues/758
https://github.com/canonical-web-and-design/base-squad/issues/759

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,13 +1,38 @@
...
-    {% elif query %}
-        <h1>Assets containing '{{ query }}'</h1>
-        <p>No assets found.</p>
-    {% endif %}
+    <div class="row">
+        <div class="col-12">
+        <form action="/" class="p-form p-form--inline">
+            <div class="p-form__group">
+                <div class="p-form__control u-clearfix">
+                    <input type="search" class="p-form__control" name="q" placeholder="Search" value="{{ query }}">
```
Comment (anthonydillon) [inline templates/index.html]: You should use the search pattern: https://docs.vanillaframework.io/patterns/search-box/

File: templates/update.html [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
+    <form method="post">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+        {% if asset.image %}
+        <a href="{{ asset.url }}" target="_blank">
+            <img class="p-image--bordered" src="{{ asset.url }}" style="max-width: 100px; max-height: 100px;">
+        </a>
+        {% endif %}
+        <label for="input-tags">Tags:</label>
+        <input id="input-tags" placeholder="e.g.: button, ubuntu" value="{{ asset.tags }}" type="text" name="tags" />
+        <button type="submit">Submit</button>
```
Comment (anthonydillon) [inline templates/update.html]: Save instead of Submit?

File: templates/index.html (hunk 1 above)
Comment (jpmartinspt) [reply templates/index.html]: Can I use that pattern and have another input in-line?

Comment (jpmartinspt) [reply templates/index.html]: Aaaaaaant help me frontend!

Comment (anthonydillon) [reply templates/index.html]: Does the search button search and filter at the same time?

Comment (jpmartinspt) [reply templates/index.html]: Yeah, just sends an empty `type` before anything is set.

**Pull request merged**